apps/internal-tool/.env.development

```
@@ -1,6 +1,11 @@
 NEXT_PUBLIC_HEXCLAVE_API_URL=http://localhost:${NEXT_PUBLIC_HEXCLAVE_PORT_PREFIX:-81}02
 NEXT_PUBLIC_HEXCLAVE_PROJECT_ID=internal
-NEXT_PUBLIC_HEXCLAVE_PUBLISHABLE_CLIENT_KEY=this-publishable-client-key-is-for-local-development-only
-NEXT_PUBLIC_HEXCLAVE_DASHBOARD_URL=http://localhost:${NEXT_PUBLIC_HEXCLAVE_PORT_PREFIX:-81}01
 NEXT_PUBLIC_SPACETIMEDB_HOST=ws://localhost:${NEXT_PUBLIC_HEXCLAVE_PORT_PREFIX:-81}39
-NEXT_PUBLIC_SPACETIMEDB_DB_NAME=stack-auth-llm
+NEXT_PUBLIC_SPACETIMEDB_DB_NAME=hexclave-ai-analytics
+# Optional in dev: set to run the automated/retried LLM QA reviews locally
+HEXCLAVE_OPENROUTER_API_KEY=
+# Dev-only server key for the seeded internal project (matches the e2e/dev default)
+HEXCLAVE_SECRET_SERVER_KEY=this-secret-server-key-is-for-local-development-only
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HEXCLAVE_SECRET_SERVER_KEY key should go before the NEXT_PUBLIC_HEXCLAVE_API_URL key

(UnorderedKey)